drivers/modem/modem_context.c

```diff
@@ -0,0 +1,185 @@
+/** @file
+ * @brief Modem context helper driver
+ *
+ * A modem context driver allowing application to handle all
+ * aspects of received protocol data.
+ */
+
+/*
+ * Copyright (c) 2019 Foundries.io
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(modem_context, CONFIG_MODEM_LOG_LEVEL);
+
+#include <kernel.h>
+
+#include "modem_context.h"
+
+static struct modem_context *contexts[CONFIG_MODEM_CONTEXT_MAX_NUM];
+
+void modem_context_hexdump(const char *desc, const u8_t *packet, size_t length)
+{
+#if defined(CONFIG_MODEM_CONTEXT_VERBOSE_DEBUG)
+	char output[sizeof("xxxxyyyy xxxxyyyy")];
```

Comment:
Why cannot we just use `LOG_HEXDUMP_DBG()` here?

Reply:
Mainly because I forgot it existed!  Fixing.